server/.classpath

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The path `/Users/sabinapokhrel/OneDrive...` won't be accessible to other users.
I wonder if this file is useful.